storefront-nextjs/.env.local.example

```
@@ -12,8 +12,9 @@ TENANT_SUBDOMAIN=demo
 # TENANT_ID=your-tenant-id
 
 # Optional - Analytics
-NEXT_PUBLIC_GA_ID=
+NEXT_PUBLIC_GA_ID=G-MLKYLX3M62
 NEXT_PUBLIC_GTM_ID=
+GOOGLE_SITE_VERIFICATION=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GOOGLE_SITE_VERIFICATION key should go before the NEXT_PUBLIC_GA_ID key

(UnorderedKey)